client/packages/sdk/.prettierrc

```diff
@@ -1,3 +1,6 @@
 {
+  "tabWidth": 2,
+  "useTabs": false,
+  "singleQuote": true,
```

Comment:
```suggestion
  "singleQuote": false,
```

Reply:
😇

Reply:
We already have 2 or 3 other projects who use single-quote with the prettier config. And the most widely-used standard in JS is single-quote.
I don't rly care so much about which quotes we use, but rather that we stick to industry standards and also across projects.
I'll put double quotes for this one, if you prefer.